Add "How Promptless uses AI" security page (#810)
Promptless/docs: src/content/docs/docs/security/how-promptless-uses-ai.mdx

---
title: How Promptless uses AI
description: "How Promptless uses AI to generate documentation, including its default and enterprise model options, no training on customer data, real-time processing without retention, and self-hosting."
slug: docs/security/how-promptless-uses-ai
type: reference
tags:
  - security
  - reference
timestamp: "2026-08-03T14:20:00Z"
sidebar:
  hidden: false
  order: 37
  label: How Promptless uses AI
---

> Promptless uses AI language models to generate documentation suggestions for your team. This page consolidates how Promptless uses AI and how it protects customer data throughout that process. Each point links to the security page where it is documented in full.

Reviewer: ```suggestion
Promptless uses AI language models to generate documentation suggestions for your team. This page explains how Promptless uses AI and how it protects customer data throughout that process. Each point links to the security page where it is documented in full.
```

## Which AI models Promptless uses

> The default configuration for cloud-hosted customers uses AWS Bedrock. Promptless is built on a model-agnostic architecture that works with a variety of models, including Anthropic, OpenAI, and open-source providers. The final choice of model is up to you. For the full list of AI infrastructure providers, see [subprocessors](/docs/security/subprocessors).

Reviewer: ```suggestion
The default configuration for cloud-hosted customers uses AWS Bedrock. Promptless is built on a model-agnostic architecture that works with a variety of models, but the final choice of model will be determined by you and the Promptless team during onboarding. For the full list of AI infrastructure providers, see [subprocessors](/docs/security/subprocessors).
```
